src/nimblepkg/nimscriptwrapper.nim

```diff
@@ -37,30 +37,19 @@ proc writeExecutionOutput(data: string) =
   display("Info", data)
 
 proc execNimscript(
-  nimsFile, projectDir, actionName: string, options: Options, isHook: bool
+  nimsFile, projectDir, actionName: string, options: Options, isHook: bool,
+  nimbleFile = ""
```

Comment:
Why a default here? Doesn't every invokation of this have a ``nimbleFile`` to supply? and shouldn't it always be supplied?

Reply:
I originally thought that `execNimscript` was meant was a nimscript executing wrapper for any nimscript file. Turns out that it was just meant to execute nimble scripts. I don't know why this don't just take a nimble file instead, every invocation I found is in the form of `getNimsFile` then `execNimscript`.

Reply:
Nim doesn't like `.nimble` files, it has to be `.nims`. Further, we add some stuff to the `.nims` file generated to retain backward compatibility.

https://github.com/nim-lang/nimble/blob/master/src/nimblepkg/nimscriptwrapper.nim#L117

Reply:
I know, what I mean is that this step can be done in `execNimscript` instead of a separated `getNimsFile` proc because `execNimscript` can only ever execute scripts created by `getNimsFile`.

Reply:
`getIniFile()` also uses it, was one way to reuse.

Reply:
So, can we resolve this?